src/HealthChecks.Rabbitmq/README.md

```diff
@@ -45,47 +26,38 @@ public void ConfigureServices(IServiceCollection services)
             var factory = new ConnectionFactory
             {
                 Uri = new Uri("amqps://user:pass@host/vhost"),
-                AutomaticRecoveryEnabled = true
             };
-            return  factory.CreateConnection();
+            return factory.CreateConnectionAsync().GetAwaiter().GetResult();
```

Comment:
Since this instance is getting registered in DI, and DI doesn't allow for async factory methods, this has to block the current thread. It won't cause problems in normal web applications, given that it will only block at start up to get the connection the first time. Once the connection is established, it will be cached as a singleton and won't block subsequent threads in the application. See https://github.com/dotnet/aspire/issues/565